src/rules/noDuplicatedBranchesRule.ts

```diff
@@ -136,9 +133,14 @@ class Walker extends tslint.RuleWalker {
     return this.getSourceFile().getLineAndCharacterOfPosition(node.getEnd()).line + 1;
   }
 
-  private takeWithoutBreak(nodes: ts.NodeArray<ts.Node>) {
+  private takeWithoutBreak(nodes: ts.Node[]) {
     return nodes.length > 0 && nodes[nodes.length - 1].kind === ts.SyntaxKind.BreakStatement
       ? nodes.slice(0, -1)
       : nodes;
   }
+
+  private expandSingleBlockStatement(nodes: ts.Node[]) {
+    const child = nodes[0];
```

Comment:
no need, `child` will be `undefined` and the next check `this.isBlock(child)` will fail